investigator/investigator/common.py

```diff
@@ -43,7 +43,11 @@ def wait_for_limit(func):
         def inner_func1(*args, **kwargs):
             total_pending = inf
             while total_pending > limit:
-                asyncio.sleep(0.5)
+                #####################################################################################################
+                # to make this awaitable would require a major rework because of the number of sync functions which #
+                # call submit_workflow.                                                                             #
+                #####################################################################################################
+                sleep(0.5)
```

Comment:
Can this be a parameter supplied via env var? Also, 0.5 seems to be too often to me.